### DISCUSSION TITLE
What is the best way to get all the routes and components of a svelte codebase?

### DISCUSSION THREAD

**Discussion by switchupcb** [sveltejs/kit#8509] [Q&A] [answered]
For example, you can find all routes by searching the `routes` folder and ignoring files that start with `__`. Is this the best way to identify all routes? Is there a way to identify all components?

**Comment by teemingc** [comment, marked as answer]
You can search for `+page` to find routes and maybe some kind of search with .svelte that also filters out the +page and +layout files so it only shows components
